### PR TITLE
Show warning on failure to delete profile files

### PR DESCRIPTION
**Pull request by initramfs**
This PR adds warnings if Qt was unable to successfully delete a profile file (.tox, .ini, .db, etc). Potentially allowing users to perform manual deletion (or, in the case of bug reports regarding profile deletion [e.g. issue #3209], allows us to verify correct operation).

### CODE REVIEW

**Comment by PKEv** [comment]
Maybe add some message to the user - logs few looks?

**Comment by SkyzohKey** [comment]
Maybe make the string translatable ?

**Comment by initramfs** [comment]
@PKEv @SkyzohKey New error dialog box now appears upon failure to delete any file, prompting the users to manually delete files. The strings are made translatable.

File: src/widget/form/profileform.cpp [hunk 1]
```diff
@@ -361,7 +361,12 @@ void ProfileForm::onDeleteClicked()
                 tr("Are you sure you want to delete this profile?", "deletion confirmation text")))
     {
         Nexus& nexus = Nexus::getInstance();
-        nexus.getProfile()->remove();
+
+        if(!nexus.getProfile()->remove())
+        {
+            GUI::showError(tr("Files could not be deleted!"), tr("Some files could not be deleted, please manually remove them."));
```
Comment (sudden6) [inline src/widget/form/profileform.cpp]: How should the user know which files to delete?

Comment (initramfs) [inline src/widget/form/profileform.cpp]: @sudden6 Well, they could check the log but I see your concern. Should the prompt show the names of the file that could not be deleted? Becauses chances are, unless the user manually tampered with the filesystem, either all will delete or none will.

Comment (sudden6) [inline src/widget/form/profileform.cpp]: > Should the prompt show the names of the file that could not be deleted?

Maybe you could list all filenames that couldn't be deleted at once? Might produce a space problem in the error message tough...

**Comment by initramfs** [comment]
@sudden6 Now shows the filenames that failed to delete in the prompt.

Example below:

Failed to delete dialog

**Pull request merged**